app/views/patients/_abortion_information.html.erb

```diff
@@ -35,6 +37,16 @@
         <% end %>
       </div>
     </div>
+
+    <div class="row">
+      <div class="col-sm-12" id="clinic_finder_container">
+
+        <%= link_to display_clinic_finder_path, class: 'display_button', remote: true do %>
```

Comment:
IIRC in the designs, this was triggerable by a link/block/button-ish deal that would open -

![screen shot 2017-08-05 at 6 20 42 pm](https://user-images.githubusercontent.com/3866868/28999164-cde42416-7a0a-11e7-912e-15f190b2f1e3.png)

Bonus if we follow that design!